test/params/mqpar1.5.3.30_MBR_sel.json

```diff
@@ -1,8 +1,8 @@
 {
-    "software_name": null,
+    "software_name": NaN,
```

Comment:
null became NaN. But I think it would be good to put MaxQuant as a name anyways.